src/test/java/com/rybka/command/ConvertCommandTest.java

```diff
@@ -0,0 +1,56 @@
+package com.rybka.command;
+
+import com.rybka.model.CurrencyData;
+import com.rybka.model.CurrencyHistory;
+import com.rybka.repository.CurrencyHistoryRepository;
+import com.rybka.service.exchange.ExchangeService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Scanner;
+
+import static org.mockito.Mockito.*;
+
+@ExtendWith(SpringExtension.class)
+public class ConvertCommandTest {
+    @Mock
+    private Scanner scanner;
+    @Mock
+    private ExchangeService exchangeService;
+    @Mock
+    private CurrencyHistoryRepository currencyHistoryRepository;
+    @InjectMocks
+    private ConvertCommand command;
+
+    @Test
```

Comment:
What about test to verify correct exception handling?

Reply:
fixed